src/NServiceBus.Transport.RabbitMQ/Administration/ManagementApi/Models/Queue.cs

```diff
@@ -67,6 +67,9 @@ public int GetDeliveryLimit()
             limit = 0;
         }
 
-        return limit ?? -1;
+        return limit ?? UnlimitedDeliveryLimitIndicatorValue;
     }
+
+    public const int UnlimitedDeliveryLimitIndicatorValue = -1_000;
+    public const int BigValueInsteadOfActuallyUnlimited = 100_000;
```

Comment:
I know we are trying to be as limited as possible since this is a "bug" fix, however, should we consider letting the user configure this? I can see an argument for not doing so, in that NSB should never reach this number of retries. This number being a "problem" presupposes a bug in RabbitMQ transport or NServiceBus core, or misconfigured recoverability settings.